tests/test_invenio_communities.py

```diff
@@ -145,3 +146,22 @@ def test_model_featured_community(app):
             start_date=t1 + timedelta(days=2)) is comm1
         assert FeaturedCommunity.get_featured_or_none(
             start_date=t1 + timedelta(days=4)) is comm2
+
+
+def test_oaipmh_sets(app):
+    """Test the OAI-PMH Sets creation."""
+    with app.app_context():
+        # Create a user
+        user1 = create_test_user()
+        comm1 = Community(id='comm1',
+                          id_user=user1.id,
+                          title='Title1',
+                          description='Description1')
+        db.session.add(comm1)
+        db.session.commit()
```

Comment:
@krzysztof there is a study material for you http://docs.sqlalchemy.org/en/rel_0_9/orm/session_basics.html.

> the `Session` establishes all conversations with the database and represents a “holding zone” for all the objects which you’ve loaded or associated with it during its lifespan